app/actions/AuthorisedAction.scala

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Do we need this file? The repo already has an AuthorisedAction defined at controllers.predicates. We should probably just be using this existing one